docs/azure_arc_jumpstart/azure_arc_k8s/day2/cluster_api/cluster_api_osm_extension/_index.md

```diff
@@ -0,0 +1,141 @@
+
+    ![Screenshot parameter examples](./03.png)
+
+ > **Note: The extra dot is due to the shell script has an *export* function and needs to have the vars exported in the same shell session as the rest of the commands.**
+
+   The script will:
+
+* Login to your Azure subscription using the SPN credentials
+* Add or Update your local _connectedk8s_ and _k8s-extension_ Azure CLI extensions
+* Create Open Service Mesh k8s extension instance
+* Create Azure Monitor k8s extension instance
+* Download and install OSM cli locally
+* Create four namespaces in kubernetes to deploy a test app
+* Onboard the Namespaces to the OSM Mesh and enable sidecar injection on the namespaces
+* Enable metrics for pods belonging to app namespaces
+* Update the namespaces to be monitored by modifying the configmap provided by the OSM
+* Deploy the apps to the namespaces
+
+You can now see that Open Service Mesh & Azure Monitor for containers extensions are now enabled in the extension tab section of the Azure Arc enabled Kubernetes cluster resource in Azure.
+
+![Screenshot extension deployment security tab](./04.png)
+
+* You can also verify the deployment by running the below _kubectl_ commands and see the deployed artifacts in the _arc-osm-system_ namespace.
+
+![Screenshot extension deployment on cluster](./05.png)
+
+* It can take ~15 minutes before you will be able to verify the integration and monitoring insights coming from OSM to Azure Monitor by following the below steps in the Azure portal.
+
+  * Verify the namespaces are showing up in the insights section of the Arc enabled Kubernetes resource in Azure portal.
+
+![Show the namespaces in the Container Insights](./06.png)
+
+  * Verify if monitoring working correctly by query the logs for pulling the data from the InsightsMetrics schema.
+  
+```shell
+InsightsMetrics | where Name contains "envoy" | extend t=parse_json(Tags) | where t.app == "bookstore"
+![Show the log analytics query ](./07.png)
+
+You can also navigate to the "Reports" tabs in the Insights section and see the OSM dashboard report got added.
+
+> **Disclaimer: This feature is under preview and in order to view the OSM report template you have to access the portal with a query string _"?feature.includePreviewTemplates=true"_ or directly accessing the Azure portal using the following [link](https://aka.ms/azmon/osmux).**
+
+![Show the report templates for OSM in the Container insights](./08.png)
+
+### Delete extension instances
+
+The following command only deletes the extension instances, but doesn't delete the Log Analytics workspace. You can also delete the extensions from the Azure Portal under the extensions section of Azure Arc enabled Kubernetes cluster resource.
+
+```bash
+az k8s-extension delete --cluster-type connectedClusters --cluster-name <name of the cluster> --resource-group <name of the resource group> --name <name of the extension> -y
+```
```

Comment:
```suggestion
The following command only deletes the extension instances but doesn't delete the Log Analytics workspace. 

```bash
az k8s-extension delete --cluster-type connectedClusters --cluster-name <name of the cluster> --resource-group <name of the resource group> --name <name of the extension> -y
```

You can also delete the extensions from the Azure Portal under the extensions section of Azure Arc enabled Kubernetes cluster resource.

```